astra-db-java/src/main/java/com/datastax/astra/client/Collection.java

```diff
@@ -415,19 +415,19 @@ public final InsertOneResult insertOne(T document) {
 
     /**
      * Asynchronously inserts a single document into the collection. This method provides the same functionality as
-     * {@link #insertOne(T document)}, but it operates asynchronously, returning a {@link CompletableFuture} that
+     * {@link #insertOne(Object)}, but it operates asynchronously, returning a {@link CompletableFuture} that
```

Comment:
Javadoc generator complained about failed reference; this seemed to fix it. Same for all other cases I changed; did not change anything else.